proto/connector_service.proto

```diff
@@ -143,7 +136,6 @@ message GetEventStreamRequest {
   }
 
   oneof request {
```

Comment:
Are we going to more request kinds to this `oneof`?

Reply:
For source, it is a one directional streaming grpc. So there should not need the `oneof` after this refactor.